doc/nrf/gs_modifying.rst

```diff
@@ -181,41 +281,34 @@ Configuring build types
 
 .. build_types_overview_start
 
-Build types enable you to use different sets of configuration options for each board.
-You can create several build type :file:`.conf` files per board and select one of them when building the application.
-This means that you do not have to use one :file:`prj.conf` file for your project and modify it each time to fit your needs.
+When the ``CONF_FILE`` variable contains a single file and this file follows the naming pattern :file:`prj_<buildtype>.conf`, then the build type will be inferred to be *<buildtype>*.
```

Comment:
Add a sentence here:

"Also, if file :file:`boards/<BOARD>_<build>.conf` exists in same folder as :file:`prj<build>.conf`, the result of merging :file:`prj_<build>.conf` and :file:`boards/<BOARD>_<build>.conf` is used."